Organic/AmpAdsInjector.php

```diff
@@ -43,12 +62,29 @@ public function handle() {
                 continue;
             }
 
+            // certain placement is blocked
+            if ( $rule && in_array( $key, $blockedKeys ) ) {
+                continue;
+            }
+
             $selectors = $placement['selectors'];
             $limit = $placement['limit'];
             $relative = $placement['relative'];
 
-            // certain placement is blocked
-            if ( $rule && in_array( $key, $blockedKeys ) ) {
+            // if placement's ad_type is set to 'outstream_video':
+            // inject connatix player instead of generic amp template
+            if ( $placement['adType'] == AD_TYPE::OUTSTREAM_VIDEO ) {
+
+                // get playspaceId from placement's 'connatixId'
+                $connatixId = trim( $placement['connatixId'] ) ?: '';
+                $psid = is_valid_uuid( $connatixId ) ? $connatixId : '';
```

Comment:
If `$psid` ends up being an empty string probably we should skip injections of Connatix, because it will not work either way

Reply:
[Yep, it's already done.](https://github.com/orgnc/wordpress-plugin/blob/c3ab7e40ce1c10c476fa7afe3d852941f171ebec/Organic/AmpAdsInjector.php#L113)